xAPI-Communication.md

```diff
@@ -1209,26 +1209,26 @@ a single defined document identified by "profileId". Otherwise, GET will return
 
 Example endpoint: http://example.com/xAPI/agents/profile
 
-Stores, changes, fetches, or deletes the specified profile document in the context of the specified Agent.  
+Stores, changes, fetches, or deletes the specified Profile document in the context of the specified Agent.  
```

Comment:
I don't think capitalizing these goes far enough to solve the current open issue around this, but maybe that can can solved in a separate PR.